src/components/experiment/ExperimentVisitsTable.tsx

```diff
@@ -73,15 +69,15 @@ function ExperimentVisitsTable({
       field: 'rowData.startsAt',
       render: (rowData: RowType) =>
         rowData.isRegistrationSubmitted
-          ? formatDate(rowData.startsAt)
+          ? toFormattedDateTime(rowData.startsAt)
```

Comment:
Here comes the same situation with the warning thing. If we use render inside the component it always gives a warning that should not re-render or whatever. The easiest fix is to keep columns outside of the component and without render option but instead something like this component: https://github.com/UserOfficeProject/user-office-frontend/blob/develop/src/components/call/CallsTable.tsx#L218